test/metabase/db/schema_migrations_test/impl.clj

```diff
@@ -196,11 +197,26 @@
                   (str "'Empty' application DB is not actually empty. Found tables:\n"
                        (u/pprint-to-str tables))))))
     (log/debugf "Finding and running migrations before %s..." start-id)
+    (#'mdb.setup/initialize-db! driver conn)
     (run-migrations-in-range! conn [1 start-id] {:inclusive-end? false})
-    (f
-     (fn migrate! []
-       (log/debugf "Finding and running migrations between %s and %s (inclusive)" start-id (or end-id "end"))
-       (run-migrations-in-range! conn [start-id end-id]))))
+    (letfn [(migrate
+              ([]
+               (migrate :up nil))
+              ([direction]
+               (migrate direction nil))
+
+              ([direction version]
+               (case direction
+                 :up
+                 (do
+                  (log/debugf "Finding and running migrations between %s and %s (inclusive)" start-id (or end-id "end"))
+                  (run-migrations-in-range! conn [start-id end-id]))
```

Comment:
`version` does not seem to be used here?

Reply:
yes, it's only used for downgrade.